### PR TITLE
homework 01-uliana

### PR DESCRIPTION
**Pull request by Uliana202**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
+body{
+	padding: 40px 20px;
+		font-family: sans-serif;
```
Comment (qodesmith) [inline css/style.css]: :hammer: Your code looks great. Just beware about proper indentation for best practices. Keep it all consistent :)

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	font-size: 18px;
+	border-top: 8px solid #79A6EA;
+	margin-top: 10px;
+	    color: #6C6C6C;
+} 
+
+
+body{
+	background: #79A6EA; /* For browsers that do not support gradients */
+    background: -webkit-linear-gradient(white, #79A6EA); /* For Safari 5.1 to 6.0 */
```
Comment (qodesmith) [inline css/style.css:13]: @Uliana202 I see your using vendor prefixes here. Would you be able to explain what they're used for? The code looks fine, just want to make sure you understand what's going on here. I love the gradient background. Nice touch.

File: css/style.css (hunk 1 above)
Comment (Uliana202) [inline css/style.css]: ok i will.

Thanks for your feedback

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    background: -o-linear-gradient(white, #79A6EA); /* For Opera 11.1 to 12.0 */
+    background: -moz-linear-gradient(white, #79A6EA;) /* For Firefox 3.6 to 15 */
+    background: linear-gradient(white, #79A6EA; )/* Standard syntax */
+}
+
+hr.style1 {
+	border-top: 4px double #79A6EA;
```
Comment (qodesmith) [inline css/style.css:20]: @Uliana202 First person to use a border style other than solid ;)

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    display: inline-block;
+}
+
+
+h1 {
+	color:dark-grey;
+	font-size: 70px; 
+	text-align: center;}
+
+nav {text-align: center;
```
Comment (qodesmith) [inline css/style.css]: :hammer: The code get's a bit difficult to read here (and below) because of indentation.

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+img {width: 300px;
+float: right;
+border:5px solid #79A6EA;
+display: inline;
+padding: 5px;
+}
+
+.purple {color: purple;}
+.purple>li>a {color: blue;}
```
Comment (qodesmith) [inline css/style.css:58]: @Uliana202 Nice use of the selectors!

File: experience.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+			<a href="index.html">Home</a> |
+ 			<a href="experience.html">Robert Plant -The Experience</a> |
+ 			<a href="https://vimeo.com/search?q=led+zeppelin"  target="_blank" >Led Zeppelin on vimeo</a>
+		</nav>
+		<hr class="style1 ">
+		<h1>Robert Plant</h1>
+		<h4>The Experience</h4>
+		
+
+		<p>
```
Comment (qodesmith) [inline experience.html:21]: :hammer: These paragraphs need to be cleaned up for spacing / indentation.

File: experience.html [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+		</p>
+
+		<p>
+Plant's early blues influences included Johnson, Bukka White, Skip James, Jerry Miller, and Sleepy John Estes. Plant had various jobs while pursuing his music career, one of which was working for the major British construction company Wimpey in Birmingham in 1967 laying tarmac on roads. He also worked at Woolworth's in Halesowen town for a short period of time. He cut three obscure singles on CBS Records[14] and sang with a variety of bands, including the Crawling King Snakes, which brought him into contact with drummer John Bonham. They both went on to play in the Band of Joy, merging blues with newer psychedelic trends.
+
+		</p>
+
+		<hr class="style1 ">
+		<h4>Thank you for visiting the fanclub at Robert Plant</h4>
+
```
Comment (qodesmith) [inline experience.html:45]: :hammer: Best practices would be to use a _single_ empty line to separate sections of your code. I would remove unnecessary empty lines like here at the end.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+			<li>Legacy</li>
+			<li>Tours</li>
+			<li class="more">Discography</li>
+			<li class="extra">References</li>
+		</ol>
+	
+		<p id="theonlyone">
+			You know, people can't fall in love with me just because I'm good at what I do.
+		</p>
+
```
Comment (qodesmith) [inline index.html:51]: :hammer: As commented about, I would remove all these unnecessary empty lines.

**Comment by qodesmith** [comment]
:+1:

**Pull request closed without merging**